src/test/java/com/nokia/uwr/service/callshandler/CallsHandlerImplTest.java

```diff
@@ -0,0 +1,74 @@
+package com.nokia.uwr.service.callshandler;
+
+import com.nokia.uwr.assigntmentsalgorithm.AssignmentsAlgorithm;
+import com.nokia.uwr.connectionmanager.ConnectionManager;
+import com.nokia.uwr.model.BTS;
+import com.nokia.uwr.model.Measurements;
+import com.nokia.uwr.model.UEMeasurement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+class CallsHandlerImplTest {
+
+    private CallsHandler callsHandler;
+    private ConnectionManager connectionManager;
+    private AssignmentsAlgorithm assignmentsAlgorithm;
+
+    private Measurements measurements;
+
+    private BTS bestBts;
+
+    @BeforeEach
+    public void setup() {
+        connectionManager = mock(ConnectionManager.class);
+        assignmentsAlgorithm = mock(AssignmentsAlgorithm.class);
```

Comment:
What is the reason for mocking these classes before each test?

Reply:
Can you point the places where we need different behaviour between test methods?